sms/constructor.tentative.https.any.js

```diff
@@ -0,0 +1,60 @@
+// META: title=SMS Receiver API: Constructor
+
+'use strict';
+
+promise_test(async t => {
+  let used = false;
```

Comment:
The tests in this file are not asynchronous, so `test` and a class function will suffice.